pages/resources/team.mdx

```diff
@@ -440,3 +440,23 @@ Creates a new company under this team. Returns a [company](#company-object) obje
 | Field | Type   | Description             |
 | ----- | ------ | ----------------------- |
 | name  | string | The name of the company |
+
+<RouteHeader method="POST" url="/teams/{team.id}/identity/verification">
+  Create Team Identity Verification
+</RouteHeader>
+
+Returns a [User Identity Verification Object](/resources/user#user-identity-verification-object).
```

Comment:
```suggestion
Creates a new verification attempt for the team. Returns a [user identity verification](/resources/user#user-identity-verification-object) object on success.
```
object always outside link, and no capital (except for acronyms like SKU)
please apply this to the other endpoints too